ts/features/pn/aar/utils/detailsById.ts

```diff
@@ -0,0 +1,7 @@
+import { MessageBodyMarkdown } from "../../../../../definitions/backend/MessageBodyMarkdown";
+import { UIMessageDetails } from "../../../messages/types";
+
+export const fillerEphemeralAARMarkdown =
+  "EPHEMERAL_AAR_INVALID_MARKDOWN".repeat(3) as MessageBodyMarkdown;
+export const isAarDetailById = (message: UIMessageDetails): boolean =>
```

Comment:
See comment above on not using a fake markdown to identify the message. Consequently, this function should be removed